crates/cheatcodes/src/inspector.rs

```diff
@@ -1577,25 +1586,28 @@ impl Cheatcodes {
         interpreter: &mut Interpreter,
         ecx: &mut EvmContext<DB>,
     ) {
-        let key = try_or_return!(interpreter.stack().peek(0));
-        let target_address = interpreter.contract().target_address;
-        if let Ok(value) = ecx.sload(target_address, key) {
-            if value.is_cold && value.data.is_zero() {
-                let arbitrary_value = self.rng().gen();
-                if self.arbitrary_storage.is_copy(&target_address) {
-                    self.arbitrary_storage.copy(
-                        &mut ecx.inner,
-                        target_address,
-                        key,
-                        arbitrary_value,
-                    );
-                } else {
-                    self.arbitrary_storage.save(
-                        &mut ecx.inner,
-                        target_address,
-                        key,
-                        arbitrary_value,
-                    );
+        if interpreter.current_opcode() == op::SLOAD {
```

Comment:
can early return on this and sload below with `let ... else`

Reply:
added in https://github.com/foundry-rs/foundry/pull/8848/commits/aec556840a846074521b048271983e44906ef7a8